etc/VerticaPyLab.conf.default

```diff
@@ -21,7 +21,7 @@ VERTICA_CONTAINER_NAME=verticadb
 VERTICA_HOST_NAME=$VERTICA_CONTAINER_NAME
 VERTICA_DB_NAME=$VERTICA_CONTAINER_NAME
 DB_NAME=demo
-VERTICA_DOCKER_IMAGE=opentext/vertica-k8s:23.4.0-0-minimal # see https://hub.docker.com/r/vertica/vertica-k8s
+VERTICA_DOCKER_IMAGE=opentext/vertica-k8s:24.1.0-0-minimal # see https://hub.docker.com/r/vertica/vertica-k8s
```

Comment:
This is going to fail. The new vertica images use vcluster instead of admintools and Verticapylab is configured to used admintools.
Is there an urgent need for 24.1 on Verticapylab?

Reply:
There is no urgent need. But we have certain functions in 24.2.1 that I have been wanting to use. If we can get the latest then it will be convenient for developers/users.

Reply:
any news on that?